### DISCUSSION TITLE
missing openApi specification

### DISCUSSION THREAD

**Discussion by reisman234** [eclipse-edc/Connector#3406] [Q&A] [answered]
In an older version of the project, the generated openapi specification `/resources/openapi` was attached, but this seems to be missing now.
Where can I look it up for the respective extensions, or do I have to search through the code??

**Comment by ndr-brt** [comment, marked as answer]
You can find them on SwaggerHub, the links can be found in the documentation: https://eclipse-edc.github.io/docs/#/hands-on

However if you need, you can still generate them with the `./gradlew resolve` command
